### ISSUE TITLE
[Feat]: Add my profile to web

### ISSUE THREAD

**Issue by MonishRaman** [codeaashu/DevDisplay#303]
### What feature?

adding my github and my other deets to DevDisplay

### Add screenshots

Add screenshots

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by ishap11** [comment]
Hi @MonishRaman  , 
Thanks for your interest in contributing to DevDisplay!
Before we proceed, I noticed you added your profile to #DevDisplay but forgot to star the repo. It's like making an amazing sandwich and forgetting to take a bite! 🌟 I think our repo is feeling a little lonely without your star 🤗, so be sure to star the DevDisplay GitHub Repository.

💌 This is a mandatory step for all contributors, and it's the first priority of contribution in DevDisplay. 💌
🚀 We look forward to your contribution once these steps are completed! 🚀

Thanks! 👍🏻
